examples/layers/pubspec.yaml

```diff
@@ -10,3 +10,4 @@ dev_dependencies:
 flutter:
```

Comment:
cc @HansMuller for review of the change to pubspec.yaml; turns out some examples in layers/ use material.